newsrevenuehub/rev-engine
# DEV-86 | Enable admins to revert to a previous version of a page

Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)

This PR is initially into DEV-1186, which should be merged before this is merged. This PR branches from and presupposes DEV-1186, so provisionally setting that as base so diff will be unique to this PR.

#### What's this PR do?

Adds django-reversion-compare as dependency and configures `DonationPageAdmin` to use it, in order to make it a bit easier for admin users to compare any two page versions when considering whether or not to revert.

#### Why are we doing this? How does it help us?

Make it easy for admins to understand differences between donation page versions when considering making a change.

#### How should this be manually tested?

In Django admin, make and save one or more changes to a donation page. Then from the detail view for that donation page, click the `History` button in the top right hand corner. There, you'll see the compare interface which will allow you to see differences between any two given versions.

#### Have automated unit tests been added?

No. We just add the package and use -- no need to test.

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

No

#### Has this been documented? If so, where?

No

#### What are the relevant tickets? Add a link to any relevant ones.

[DEV-86 | Hub admin can revert to previous version of page](https://news-revenue-hub.atlassian.net/jira/software/c/projects/DEV/boards/3?modal=detail&selectedIssue=DEV-86)

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

No.

#### Are there next steps? If so, what? Have tickets been opened for them?

No.
